websites/E/e-z/presence.ts

```diff
@@ -0,0 +1,118 @@
+const presence = new Presence({
+	clientId: "1018191690210754693",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/Crfg6sn.png",
+		},
+		username = document.querySelector("p[id=premid-username]")?.textContent,
+		uid = document.querySelector("p[id=premid-uid]")?.textContent,
+		{ pathname, href, hostname } = document.location;
+
+	switch (hostname) {
+		case "e-z.host":
+			if (typeof username !== "undefined" && typeof uid !== "undefined") {
+				presenceData.details = `Username: ${username} | ${uid}`;
+				presenceData.buttons = [
+					{
+						label: "User Page",
+						url: `https://e-z.host/u/${username}`,
+					},
+				];
+			} else presenceData.details = "Not logged in";
+
+			switch (pathname) {
+				case "/":
+					presenceData.state = "Viewing the home page";
+					presenceData.buttons = [
+						{
+							label: "Discord",
+							url: "https://discord.gg/ez",
+						},
+					];
```

Comment:
This is forbidden 
![image](https://user-images.githubusercontent.com/64903135/190509072-6f679009-e659-423f-ac06-f9a9bb65ae20.png)